pkg/timeutils/json_test.go

```diff
@@ -0,0 +1,46 @@
+package timeutils
+
+import (
+	"testing"
+	"time"
+)
+
+func TestFastMarshalJSONWithInvalidDate(t *testing.T) {
+	aTime := time.Date(-1, 1, 1, 0, 0, 0, 0, time.Local)
+	json, err := FastMarshalJSON(aTime)
+	if err == nil {
+		t.Fatalf("FastMarshalJSON should throw an error, but was '%v'", json)
+	}
+	anotherTime := time.Date(10000, 1, 1, 0, 0, 0, 0, time.Local)
+	json, err = FastMarshalJSON(anotherTime)
+	if err == nil {
+		t.Fatalf("FastMarshalJSON should throw an error, but was '%v'", json)
+	}
+
+}
+
+func TestFastMarshalJSON(t *testing.T) {
+	aTime := time.Date(2015, 5, 29, 11, 0, 0, 0, time.UTC)
+	json, err := FastMarshalJSON(aTime)
```

Comment:
Can you change it so that we don't repeat the same numbers? Then we can be sure each field is used for the correct spot in the output. E.g.  `time.Date(2015, 5, 29, 11, 1, 2, 3, time.UTC)`